### PR TITLE
Release 8.0.0

### PR DESCRIPTION
**Pull request by max-rocket-internet**
Waiting on these 2 PRs to be merged:

https://github.com/terraform-aws-modules/terraform-aws-eks/pull/650
https://github.com/terraform-aws-modules/terraform-aws-eks/pull/639

Resolves https://github.com/terraform-aws-modules/terraform-aws-eks/issues/635
Resolves https://github.com/terraform-aws-modules/terraform-aws-eks/issues/628

### CODE REVIEW

**Comment by max-rocket-internet** [comment]
Some ugly output from https://github.com/terraform-aws-modules/terraform-aws-eks/pull/639

```
module.eks.aws_eks_cluster.this[0]: Still creating... [10m1s elapsed]
module.eks.aws_eks_cluster.this[0]: Still creating... [10m11s elapsed]
module.eks.aws_eks_cluster.this[0]: Provisioning with 'local-exec'...
module.eks.aws_eks_cluster.this[0] (local-exec): Executing: ["/bin/sh" "-c" "    until curl -k https://106D3ABA497FF44D1B14389CAF75A3B4.yl4.us-west-2.eks.amazonaws.com/healthz >/dev/null; do sleep 4; done\n"]
module.eks.aws_eks_cluster.this[0] (local-exec):   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
module.eks.aws_eks_cluster.this[0] (local-exec):                                  Dload  Upload   Total   Spent    Left  Speed
module.eks.aws_eks_cluster.this[0] (local-exec):   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
module.eks.aws_eks_cluster.this[0] (local-exec):   0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
module.eks.aws_eks_cluster.this[0] (local-exec):   0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
module.eks.aws_eks_cluster.this[0] (local-exec):   0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
```

File: examples/managed_node_groups/main.tf [hunk 1]
```diff
@@ -113,7 +113,6 @@ module "eks" {
         ExtraTag = "example"
       }
     }
-    defaults = {}
```
Comment (max-rocket-internet) [inline examples/managed_node_groups/main.tf]: @dpiddockcmp is it OK remove this? Without the example creates 2 node groups where I think it should be a single?

**Comment by max-rocket-internet** [comment]
OK apart from the 2 node group creation and the ugly output from `curl`, all 4 examples init, plan, apply, all nodes `Ready` and then destroy without any errors at all 🎉

File: examples/managed_node_groups/main.tf (hunk 1 above)
Comment (dpiddockcmp) [reply examples/managed_node_groups/main.tf]: Should be fine. It was for testing some of the for_each and ternary logic. Terraform gets more unpredictable with multiple items. Used to confirm the fix in #633

**Comment by max-rocket-internet** [comment]
I'm gonna ignore the Lint error for now, I think there's a fix in another PR.

**Pull request merged**